index.html

```diff
@@ -1,140 +1,176 @@
 <html>
+  <head>
+    <title>"Um, Actually..." stats</title>
+    <meta
```

Comment:
I see that your editor decided to convert everything to spaces for some reason; please keep it using tabs.  VSCode's default settings for HTML pages should do what I want.

Reply:
Oof, that's rough. The worst part is that I'm always telling people to use tabs instead of spaces for accessibility. I wonder what happened to my settings.. let me fix all this up.

Reply:
Used the default formatter; made the head element match more closely to what it was originally, though the default vscode formatter really wanted to split the viewport meta tag onto multiple lines.